drivers/ads101x/include/ads101x_params.h

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2017 OTA keys S.A.
+ *               2018 Matthew Blue <matthew.blue.neuro@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_ads101x
+ * @{
+ *
+ * @file
+ * @brief       Default configuration for ADS101x/111x devices
+ *
+ * @author      Vincent Dupont <vincent@otakeys.com>
+ * @author      Matthew Blue <matthew.blue.neuro@gmail.com>
+ */
+
+#ifndef ADS101X_PARAMS_H
+#define ADS101X_PARAMS_H
+
+#include "board.h"
+#include "saul_reg.h"
+#include "ads101x.h"
+#include "ads101x_regs.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name    Set default configuration parameters for the ADS101x/111x driver
+ * @{
+ */
+#ifndef ADS101X_PARAM_NAME
```

Comment:
This parameter is unused, I would remove it

Reply:
ok, but then you should this kind of struct:
```c
static const saul_reg_info_t ads101s_saul_info[] =
{
    ADS101X_SAUL_INFO
};
```
like it's done for (nearly) all other drivers

Reply:
I don't know why I did it that way. Fixed.